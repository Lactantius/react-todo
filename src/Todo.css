.Todo {
  display: flex;
  gap: 0.5rem;
  margin-top: 1rem;
}

.Todo button {
  color: white;
  background-color: #e86624;
  border: none;
  border-radius: 5px;
}
