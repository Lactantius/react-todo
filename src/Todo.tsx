import React from "react";
import "./Todo.css";

function Todo(): JSX.Element {
  return <div className="Todo"></div>;
}

export default Todo;
