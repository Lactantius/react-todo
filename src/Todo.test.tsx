import React from "react";
import { render } from "@testing-library/react";
import Todo from "./Todo";

it("renders", () => {
  render(<Todo text="test" id="12345" remove={() => null} />);
});

it("matches snapshot", () => {
  const { asFragment } = render(
    <Todo text="test" id="12345" remove={() => null} />
  );
  expect(asFragment()).toMatchSnapshot();
});
