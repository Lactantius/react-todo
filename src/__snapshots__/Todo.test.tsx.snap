// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Todo"
    id="12345"
  >
    <span>
      test
    </span>
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
